appsembler/figures
# Update test infrastructure to support testing DLReports

This PR updates Figures test infrastructure. This updates missing functionality and fields in mocks needed to support DLReports

Details:

- Add missing fields to UserProfile
- Regenerated student mock migration
- Added Cohorts to tests/factories
- Fixed test_admin - Missing email Test was originally broken. It should have expected user email but did not. The test did not fail before because `UserFactory` did not
previously have an email field
- Updated expected fields for tests using UserProfile serialization
- Add course_groups to devsite settings and test_settings
- Add exceptions module to xmodule modulestore mocks
